### PR TITLE
Add readme docs to helm chart

### LINKED ISSUE #491: Add metadata to helm chart: readme, changelog
would be pretty handy and give the perfect professional grade, to have the README and the changelog within the helm chart.
usecase: e.g. in deployment tools or in public places like artifacthub.i.o

## README:
function mesh: no README
https://artifacthub.io/packages/helm/function-mesh/function-mesh-operator

example with included README:
kube prometheus stack
https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack

background: https://helm.sh/docs/topics/charts/#chart-license-readme-and-notes
check with https://helm.sh/docs/helm/helm_show_readme/

## Changelog:
function mesh: no Changelog
https://artifacthub.io/packages/helm/function-mesh/function-mesh-operator

example with included CHANGELOG
Apache Airflow
https://artifacthub.io/packages/helm/apache-airflow/airflow?modal=changelog

2022-10-25_17h05_16

2022-10-25_14h16_08

### PR DESCRIPTION
**Pull request by tpiperatgod**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #491 

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

- add README.md documentation to Chart
- currently, there are no significant changes to our Chart itself. You can refer to the Operator's CHANGELOG document for changes to the Operator

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by freeznet — APPROVED** [review]
LGTM, do we need to add the docs update to the release process?

**Comment by tpiperatgod** [comment]
Yes. I'll sort out the documentation later, in the meantime, there are some code style issues here that need to be fixed.

File: charts/function-mesh-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# function-mesh-operator
+
+![Version: 0.2.7](https://img.shields.io/badge/Version-0.2.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+
+function mesh operator Helm chart for Kubernetes
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:5]:
```suggestion
The Function Mesh operator Helm chart for Kubernetes
```

File: charts/function-mesh-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+**Homepage:** <https://github.com/streamnative/function-mesh>
+
+## Maintainers
+
+| Name | Email | Url |
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:11]:
```suggestion
| Name | Email | URL |
```

File: charts/function-mesh-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    **Output**
+
+    ```
+    NAME                                               READY   STATUS    RESTARTS   AGE
+    function-mesh-controller-manager-5f867557c-d6vf4   1/1     Running   0          8s
+    ```
+
+### Verify installation
+
+- This example shows how to verify whether Function Mesh is installed successfully.
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:131]: I think this step is repeated with step 3 in the previous section. You can remove either one of them.

File: charts/function-mesh-operator/README.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+### Uninstall Function Mesh
+
+1. Use the following command to uninstall Function Mesh through Helm.
+
+> **Note**
+>
+> `${NAMESPACE}` indicates the namespace where Function Mesh Operator is installed.
+
+```bash
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:174]: The indent of the command is not correct.

File: charts/function-mesh-operator/README.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+helm delete function-mesh -n ${NAMESPACE}
+```
+
+2. Remove the Secrets that contain the signed certificate.
+
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:179]: ditto

File: charts/function-mesh-operator/charts/admission-webhook/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# admission-webhook
+
+![Version: 0.2.7](https://img.shields.io/badge/Version-0.2.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+
+function mesh admission webhook Helm chart for Kubernetes
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/charts/admission-webhook/README.md:5]:
```suggestion
The Function Mesh admission webhook Helm chart for Kubernetes
```

File: charts/function-mesh-operator/charts/admission-webhook/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+**Homepage:** <https://github.com/streamnative/function-mesh>
+
+## Maintainers
+
+| Name | Email | Url |
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/charts/admission-webhook/README.md:11]:
```suggestion
| Name | Email | URL |
```

File: charts/function-mesh-operator/charts/admission-webhook/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+| ---- | ------ | --- |
+| Function Mesh | <function-mesh@streamnative.io> | https://github.com/streamnative/function-mesh |
+
+## Source Code
+
+* <https://github.com/streamnative/function-mesh>
+
+## Values
+
+| Key | Type | Default | Description |
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/charts/admission-webhook/README.md:21]: There is no info un the "Description" column, is there any need to keep this column?

File: charts/function-mesh-operator/README.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| (Built-in) | admission-webhook | 0.2.7 |
+
+## Values
+
+| Key | Type | Default | Description |
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:27]: There is no content in the description column. Is there any need to keep this column?

File: charts/function-mesh-secrets-webhook/README.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# function-mesh-secrets-webhook
+
+![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.2.0](https://img.shields.io/badge/AppVersion-0.2.0-informational?style=flat-square)
+
+custom certificates for Function Mesh admission control webhook
```
Comment (Huanli-Meng) [inline charts/function-mesh-secrets-webhook/README.md:5]:
```suggestion
The custom certificates for Function Mesh admission control webhook
```

File: charts/function-mesh-secrets-webhook/README.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+**Homepage:** <https://github.com/streamnative/function-mesh>
+
+## Maintainers
+
+| Name | Email | Url |
```
Comment (Huanli-Meng) [inline charts/function-mesh-secrets-webhook/README.md:11]:
```suggestion
| Name | Email | URL |
```

**Review by Huanli-Meng — APPROVED** [review]
leave some comments. PTAL

**Review by nlu90 — APPROVED** [review]

**Review by jiangpengcheng — APPROVED** [review]

**Comment by tpiperatgod** [comment]
I don't think we need to pass the Codacy static code analysis, right? Because it seems a bit ambiguous and conflicts with @Huanli-Meng 's suggestions.

**Review by freeznet — APPROVED** [review]

**Comment by freeznet** [comment]
merging and ignoring the Codacy suggestions.

**Pull request merged**